src/app.jsx

```diff
@@ -51,11 +55,62 @@ export class Application extends React.Component {
         this.setState({ alerts: alerts });
     }
 
+    componentDidMount() {
+        this.subscribeToSystemd();
+        this.updateCertmongerService();
+    }
+
+    subscribeToSystemd() {
+        const systemdClient = cockpit.dbus('org.freedesktop.systemd1', { bus: "system" });
+        systemdClient.subscribe(
+            // path string source:
+            // #dbus-send --system --dest=org.freedesktop.systemd1 --type=method_call --print-reply /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager.ListUnits
+            { interface: 'org.freedesktop.DBus.Properties', path: '/org/freedesktop/systemd1/unit/certmonger_2eservice', member: 'PropertiesChanged' },
+            (path, iface, signal, args) => {
+                if (args[0] === "org.freedesktop.systemd1.Unit")
+                    this.updateCertmongerService();
+            }
+        );
+    }
+
+    updateCertmongerService() {
+        const { initialPhase } = this.state;
+
+        const certmongerService = service.proxy(CERTMONGER_SERVICE_NAME);
+        certmongerService.wait(() => {
+            if (initialPhase && certmongerService.state === "stopped") {
+                certmongerService.start()
+                        .fail(error => this.setState({ startErrorMessage: JSON.stringify(error) }));
+            }
+            this.setState({ initialPhase: false, certmongerService });
+        });
+    }
+
     render() {
+        const { certmongerService, startErrorMessage } = this.state;
+
+        const certificatesBody = (
```

Comment:
Do you also plan to do translations? But always good to at least mark string for translations

Reply:
No plans for that currently, but I agree it's good to have it marged.

Reply:
Fixed